eslint.config.js

```diff
@@ -0,0 +1,38 @@
+import { dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+import eslintPluginPrettierRecommended from 'eslint-plugin-prettier/recommended';
+
+export default tseslint.config(
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  eslint.configs.recommended,
+  eslintPluginPrettierRecommended,
+  ...tseslint.configs.recommendedTypeChecked,
+  {
+    languageOptions: {
+      parserOptions: {
+        project: ['./tsconfig.eslint.json', './packages/*/tsconfig.json'],
+        // import.meta.dirname requires node 21
+        tsconfigRootDir: dirname(fileURLToPath(import.meta.url)),
+      },
+    },
+    rules: {
+      'prettier/prettier': 2, // Means error
+      '@typescript-eslint/no-explicit-any': 'off',
+      '@typescript-eslint/ban-types': 'off',
+      '@typescript-eslint/unbound-method': 'off',
+      '@typescript-eslint/no-unsafe-assignment': 'off',
+      '@typescript-eslint/ban-ts-comment': 'off',
+      '@typescript-eslint/no-redundant-type-constituents': 'off',
+      '@typescript-eslint/no-duplicate-type-constituents': 'off',
+      '@typescript-eslint/no-unsafe-argument': 'off',
+      '@typescript-eslint/no-floating-promises': 'off',
+      '@typescript-eslint/no-unsafe-return': 'off',
+      '@typescript-eslint/no-misused-promises': 'off',
+      '@typescript-eslint/require-await': 'off',
+      '@typescript-eslint/no-unused-vars': 'off',
+      '@typescript-eslint/no-empty-object-type': 'off',
```

Comment:
Would be nice to start fixing those

Reply:
yes, had to add those as they came with the latest update of `typescript-eslint`. will address this in separate PRs